### DISCUSSION TITLE
Issue on Localization

### DISCUSSION THREAD

**Discussion by KannanKrish** [OrchardCMS/OrchardCore#7680] [Q&A]
I followed the setup https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-5.0

I have asp.net core 5 projects. In all module, I want to use RESX localization functionality.  How to setup?

But not working. How to resolve this issue.

**Comment by hishamco** [comment]
FYI @KannanKrish OC disabled RESX localization by default and uses Portable Objects instead, if you want to bring REX back, you need to register `ResourceManagerStringLocalizer` to DI
